nixos/modules/services/misc/mautrix-telegram.nix

```diff
@@ -1,23 +1,23 @@
-{ config, pkgs, lib, ... }:
+{ config, pkgs, lib, utils, ... }:
```

Comment:
What's `utils` for?

Reply:
Oh, nothing. I tried to use `utils.genJqSecretReplacementSnippet`, but it didn't work out.